tests/appsec/waf/test_blocking.py

```diff
@@ -24,7 +58,11 @@ def assert_valid_html_blocked_template(body: str) -> None:
         _read_file(BLOCK_TEMPLATE_HTML_MIN_V2),
     }
 
-    assert body in valid_templates
+    # Check for v3 template with dynamic block_id
+    normalized_body = _normalize_html_block_id(body).rstrip()
+    normalized_v3 = _normalize_html_block_id(_read_file(BLOCK_TEMPLATE_HTML_MIN_V3)).rstrip()
```

Comment:
Why do you need to normalise the template? It seems to already contain the right ID and var name